src/main/resources/elasticsearch/addOnInfoAndVersions-mappings.json

```diff
@@ -14,7 +14,8 @@
         "type": "text"
       },
       "tags": {
-        "type": "keyword"
+        "type": "keyword",
+	"index": "not_analyzed"
```

Comment:
Just check and turns out I was wrong. Don't know why it happened before but now just the type keyword is sufficient . I have changed it